### ISSUE TITLE
Likes: Adding Likes to comments

### ISSUE THREAD

**Issue by lschuyler** [Automattic/jetpack#3224]
Is it possible to add Likes to comments?

Requested in 2478187-t

cc: @beaulebens

**Comment by beaulebens** [comment]
Right now, this is not possible unfortunately, due mainly to performance problems.

We looked into porting the implementation we have on WordPress.com over to Jetpack, and went down the path of doing it similar to how Post Likes work. On a technical level, they are displayed in an iframe for a number of reasons, with the biggest one being around making sure that people can't artificially create Likes (by using JavaScript to just trigger "liking" a post as soon as someone visits it).

Unfortunately, using this approach for Comment Likes gets ugly very quickly if you have a lot of comments on a page, and try to add an iframe for each one. You very quickly end up with a sluggish browser, and in some cases you see the browser actually crash when it overloads with iframes. Even if it doesn't cause direct problems, you'll see a massive flood of requests (one for each comment) going out to WP.com to load the iframe for each comment, which is pretty terrible for the overall performance of a site.

So in short; until we can figure out a way to have them injected directly into the page, but still protect the integrity of the data, we can't ship Comment Likes in Jetpack.

**Comment by jeherve** [comment]
Also suggested here:
https://wordpress.org/support/topic/adding-likes-to-comments

**Comment by aheckler** [comment]
2983685-t

**Comment by aheckler** [comment]
3096809-t

**Comment by seejacobscott** [comment]
3115874-t
